k8s/clusters/auto-image-update/bases/image-repo-policies/add-easm-assets-to-tracker-image-repo-policy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImageRepository
+metadata:
+  name: add-easm-assets-to-tracker
+  namespace: flux-system
+spec:
+  image: gcr.io/track-compliance/add-easm-assets-to-tracker
```

Comment:
We've migrated away from the `gcr.io` repo to `northamerica-northeast1-docker.pkg.dev/track-compliance/tracker`